src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+xref: OMIA:001411 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001411"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019780 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog anotia
+
+[Term]
+id: MONDO:1010091
+name: microtia, non-human animal
+def: "Microtia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001952 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001952"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0010920 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog microtia
+
+[Term]
+id: MONDO:1010092
+name: AA amyloidosis, non-human animal
+def: "AA amyloidosis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000038 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000038"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019439 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog AA amyloidosis
+
+[Term]
+id: MONDO:1010093
+name: AL amyloidosis, non-human animal
+def: "AL amyloidosis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000039 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000039"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019438 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog AL amyloidosis
+
+[Term]
+id: MONDO:1010094
+name: analphalipoproteinaemia, non-human animal
+def: "Analphalipoproteinaemia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000042 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000042"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000042"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008783 {source="OMIA:000042"} ! cross-species analog Tangier disease
+
+[Term]
+id: MONDO:1010095
+name: citrullinaemia, non-human animal
+def: "Citrullinaemia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000194 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000194"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000194"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008988 {source="OMIA:000194"} ! cross-species analog citrullinemia type I
+
+[Term]
+id: MONDO:1010096
+name: glycogen storage disease Ia, non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.